### ISSUE TITLE
Installation of 1.1.3 failed

### ISSUE THREAD

**Issue by FireWizard52** [andrei-tatar/node-red-contrib-smartnora#74]
Hi,

I tried to install the latest version of SmartNora (1.1.3) Neither the installation from the "Palette", nor from de command line succeeded. An earlier install ended successfully in a FreeBSD jail .

Hardware Raspberry Pi 1B
Node Red: 2.0.5
Node: V12.22.1
Npm: V6.14.12
Upgraded from Smartnora v 1.1.0

In all cases it ends with:

npm install node-red-contrib-smartnora
npm ERR! Unexpected end of JSON input while parsing near '...xoSf6A==","shasum":"c'

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2021-08-09T19_56_55_095Z-debug.log

The Debug log below:

[debug.log](https://github.com/andrei-tatar/node-red-contrib-smartnora/files/6957181/debug.log)

Any idea, what could cause this.

Regards

**Comment by andrei-tatar** [comment]
It would seem that you have some corrupted files in your node_modules directory. When this happened to me, it was the SD card.
You can try:
`rm -rf node_modules` followed by `npm install`.

**Comment by andrei-tatar** [comment]
There also this issue, where a user reported he wasn't able to install on RPi 1: https://github.com/andrei-tatar/node-red-contrib-smartnora/issues/29

I personally don't have a RPi 1 to test with...

**Comment by FireWizard52** [comment]
Hi,

Problem solved.

I started with:

> npm cache clean --force`

Afterwards I tried to install smartnora from the command line.

> npm install node-red-contrib-smartnora@1.1.3

This resulted that the ssh connection closed and I was unable to log in again. So the only way to get in again was to power down the Pi. I do not like that, but there was no alternative.

After starting up it turned out that version 1.1.3 had not been installed and still v1.1.0. was the "active"one.
Another install was succesful and now v 1.1.3 is running fine.

It has been solved, but I still don't now the real cause. We will see what will happen.

Thanks anyhow @andrei-tatar for your response.

Regards
